recipes-debian/vlan/vlan_debian.bb

```diff
@@ -0,0 +1,25 @@
+# base recipe: meta-openembedded/meta-networking/recipes-connectivity/vlan/vlan_1.9.bb
+# base branch: warrior
+# base commit: a24acf94d48d635eca668ea34598c6e5c857e3f8
+
+inherit debian-package
+require recipes-debian/sources/vlan.inc
+
+SUMMARY = "VLAN provides vconfig utility"
+HOMEPAGE = "http://www.candelatech.com/~greear/vlan.html"
+SECTION = "misc"
+LICENSE = "GPLv2"
```

Comment:
Please see the "Copyright File" from the following page.
https://packages.debian.org/buster/misc/vlan

The file says this is GPLv2+.